src/config/dynamic-theme-fixes.config

```diff
@@ -6129,6 +6129,20 @@ INVERT
 
 ================================
 
+gorod.mos.ru
+
+INVERT
+.scene__logo
+div[id^="map"] > div > div > div
+div[style^="text-align: center; margin"] > div > div:first-of-type
+
+CSS
+#footer {
+    background: #162F29 !important;
```

Comment:
Could this be in the style of `${}`?

Reply:
And what is it for? And like `${#CCECE8}` is fine.

Reply:
> And what is it for?

To ensure that different user settings will get the correct dynamic color.


> And like `${#CCECE8}` is fine.

Yap that's fine.